### PR TITLE
Disable project exporting options when Use Max Procedural Maps turned on (issue #175)

### PR DESCRIPTION
**Pull request by usakhelo**
Scene export ui elements are disabled when Use Max Procedurals mode is selected.

### CODE REVIEW

File: src/appleseed-max-impl/appleseedrenderer/appleseedrendererparamdlg.cpp [hunk 1]
```diff
@@ -562,6 +565,9 @@ namespace
 
         virtual void init(HWND hwnd) override
         {
+            m_radio_render_only = GetDlgItem(hwnd, IDC_RADIO_RENDER);
```
Comment (dictoon) [inline src/appleseed-max-impl/appleseedrenderer/appleseedrendererparamdlg.cpp]: We should always strive to keep orderings consistent, everywhere. Here, ordering of member declarations and ordering of initializations should match.

File: src/appleseed-max-impl/appleseedrenderer/appleseedrendererparamdlg.cpp [hunk 2]
```diff
@@ -583,18 +589,22 @@ namespace
             m_spinner_scale_multiplier->SetResetValue(RendererSettings::defaults().m_scale_multiplier);
             m_spinner_scale_multiplier->SetValue(m_settings.m_scale_multiplier, FALSE);
 
-            enable_disable_controls();
+            enable_disable_controls(false);
         }
 
-        void enable_disable_controls()
+        void enable_disable_controls(bool use_max_procedural_maps)
```
Comment (dictoon) [inline src/appleseed-max-impl/appleseedrenderer/appleseedrendererparamdlg.cpp]: Make arg const.

File: src/appleseed-max-impl/appleseedrenderer/appleseedrendererparamdlg.cpp [hunk 3]
```diff
@@ -563,6 +566,9 @@ namespace
         virtual void init(HWND hwnd) override
         {
             m_static_project_filepath = GetDlgItem(hwnd, IDC_STATIC_PROJECT_FILEPATH);
```
Comment (dictoon) [inline src/appleseed-max-impl/appleseedrenderer/appleseedrendererparamdlg.cpp]: `m_static_project_filepath` and `m_text_project_filepath` should probably stay together.

Comment (usakhelo) [reply src/appleseed-max-impl/appleseedrenderer/appleseedrendererparamdlg.cpp]: I'll move it here and change declaration order too...

**Pull request merged**